packages/components-js/tests/a11y/specs/axe-core/common/components.a11y.ts

```diff
@@ -0,0 +1,83 @@
+import { type Page } from '@playwright/test';
+import { test, expect } from '../../../helpers';
+import { setupScenario } from '../../../../vrt/helpers';
+import { TAG_NAMES, type TagName } from '@porsche-design-system/shared';
+import { viewportWidthXXS, viewportWidthM, themes } from '@porsche-design-system/shared/testing/playwright.vrt';
+import { getComponentMeta } from '@porsche-design-system/component-meta';
+
+const components = (TAG_NAMES as unknown as TagName[])
+  .filter((tagName) => {
+    // TODO: should not needed to be maintained like this, e.g. find a logic here with matching names or use/extend getComponentMeta() accordingly
+    return !/item$|-table-|-select-wrapper-|multi-select-option|select-option$/.test(tagName);
```

Comment:
Is this a blacklist? If so could you create a variable so it will be more clear?